src/test/java/de/medizininformatikinitiative/torch/service/CrtdlValidatorServiceTest.java

```diff
@@ -70,19 +70,40 @@ void referenceWithoutLinkedGroups() throws ValidationException {
     }
 
     @Test
-    void validInput() throws ValidationException {
-        Crtdl crtdl = new Crtdl(node, new DataExtraction(List.of(patientGroup, new AttributeGroup("test", "https://www.medizininformatik-initiative.de/fhir/core/modul-labor/StructureDefinition/ObservationLab", List.of(), List.of(new Filter("token", "code", List.of(new Code("some-system", "some-code"))))))));
+    void validInput_withoutFilter() throws ValidationException {
```

Comment:
I like to have only one AG here, the `patientGroup`, the one without filter. The test is way to complex with two AG's.